server/utils/mailer.js

```diff
@@ -7,7 +8,7 @@ const sendEmail = async (to, msgKey, msgParams = {}, lang) => {
   const mailSubject = i18n.t(`${msgKey}.subject`)
   const msgBody = i18n.t(`${msgKey}.body`, msgParams)
 
-  await aws.sendEmail(process.env.ADMIN_EMAIL, to, mailSubject, msgBody)
+  await aws.sendEmail(ProcessUtils.ENV.adminEmail, to, mailSubject, msgBody)
```

Comment:
do you mean ProcessEnv.adminEmail?

Reply:
I also didn't see that we need to specify ADMIN_EMAIL in .env @sgiaccio and neither we were notified. @fefrain could you add it to the README, thanks.

Reply:
ok